moveit_core/robot_model/include/moveit/robot_model/fixed_joint_model.h

```diff
@@ -52,7 +52,7 @@ class FixedJointModel : public JointModel
   void getVariableRandomPositions(random_numbers::RandomNumberGenerator& rng, double* values,
                                   const Bounds& other_bounds) const override;
   void getVariableRandomPositionsNearBy(random_numbers::RandomNumberGenerator& rng, double* values,
-                                        const Bounds& other_bounds, const double* near,
+                                        const Bounds& other_bounds, const double* _near,
```

Comment:
Could you split this commit into smaller more descriptive ones and describe why strange prefixes are needed here?

Reply:
Apparently `far` and `near` used to be keywords: https://stackoverflow.com/questions/8948493/visual-studio-doesnt-allow-me-to-use-certain-variable-names

We can also add `#undef near` to the beginning of this file. Do you prefer that?

Reply:
oh wow, they never cease to impress me ... but who includes `windows.h` here?

Reply:
maybe rostime or some other dependency? Maybe this was also done by replacing all occurrences of near globally.
We can try to keep the patches more minimal for upstream ... now that we have a working CI for Windows.

Reply:
I'd prefer a
```
#if  msvc
#undef near
#endif
```
here or as close to the pollutant yes

Reply:
What's the big deal with changing a variable name?
`near` is a bad name for the value anyway and I'd prefer to just change it to `seed` or `mean`.

We should not add MS specific code unless that is really needed.

Reply:
completely up to you, happy to make either change.

Reply:
I would prefer renaming this variable to something else instead of the underscore prefix.

Reply:
you guys just have to tell me which name to choose?

Should I go with `seed`?

Reply:
Yes, just use `seed` please.

Reply:
rename to seed done.